pkg/ldapdn/ldapdn.go

```diff
@@ -0,0 +1,36 @@
+package ldapdn
+
+import (
+	"github.com/go-ldap/ldap/v3"
+	"golang.org/x/text/cases"
+)
+
+// While formally some RDN attributes could be casesensitive
+// maybe we should just skip the DN parsing and just casefold
+// the entire DN string?
+func NormalizeParsedDN(dn *ldap.DN) string {
```

Comment:
The API of this module stutters a bit. None of the functions should have "DN" in their name as this is already indicated by the package "ladppn". So i sugget to rename this function to `Normalize`.

Reply:
Makes sense. Updated.